app/assets/stylesheets/common/base/tagging.scss

```diff
@@ -203,6 +203,22 @@ header .discourse-tag {
   }
 }
 
+.hashtag-autocomplete {
+  ul li a {
```

Comment:
maybe we need a class here?

Reply:
Yes good point will just add a class to the link